client/my-sites/sharing/connections/services/facebook.js

```diff
@@ -19,6 +19,10 @@ export class Facebook extends SharingService {
 		...SharingService.defaultProps,
 	};
 
+	recordLinkClickEvent = () => {
+		this.props.recordTracksEvent( 'calypso_publicize_facebook_pages_link_click' );
```

Comment:
I feel we should be more explicit about the exact event we're tracking. What about the following event name?

`calypso_publicize_facebook_profile_connection_failure_notice_link_click`